src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const changeSearch = (event: React.ChangeEvent<HTMLInputElement>) => (
+    setQuery(event.target.value)
+  );
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    movie.title.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
    movie.title.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="457" alt="image" src="https://user-images.githubusercontent.com/55802630/184603323-08f35fbc-afd0-4e33-9a5a-12ae584f4071.png">

Reply:
get it, thanks